.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -2,185 +2,193 @@ name: Bug Report
-        },
-        cache: {
-          cacheLocation: "sessionStorage"
-          storeAuthStateInCookie: false
-        }
-      }
-  validations:
-    required: true
-    
-- type: textarea
-  attributes:
-    label: Relevant Code Snippets
-    description: "Please provide relevant code snippets that can be used to reproduce this issue."
-    render: javascript
-  validations:
-    required: true
-    
-- type: textarea
-  attributes:
-    label: Reproduction Steps
-    description: "Please provide steps to reproduce or a link to a sample which demonstrates this behavior."
-    placeholder: |
-      1.
-      2.
-      3.
-  validations:
-    required: true
-    
-- type: textarea
-  attributes:
-    label: Expected Behavior
-    description: "Please describe what you expect the behavior to be."
-  validations:
-    required: true
-    
-- type: dropdown
-  attributes:
-    label: Identity Provider
-    options: 
-      - Azure AD / MSA
-      - Azure B2C Basic Policy
-      - Azure B2C Custom Policy
-      - ADFS
-      - Other
-  validations:
-    required: true
-    
-- type: dropdown
-  attributes:
-    label: Browsers Affected (Select all that apply)
-    multiple: true
-    options: 
-      - Chrome
-      - Firefox
-      - Edge
-      - Safari
-      - Internet Explorer
-      - None (Server)
-      - Other
-  validations:
-    required: true
-    
-- type: input
-  attributes:
-    label: Regression
-    description: "If this behavior worked before, please enter the last working version(s)."
-    placeholder: "@azure/msal-browser 2.0.0"
-    
-- type: dropdown
-  attributes:
-    label: Source
-    description: "Are you an internal or external customer?"
-    options: 
-      - Internal (Microsoft)
-      - External (Customer)
-  validations: 
-     required: true
-    
-- type: markdown
-  attributes:
-    value: "## Security Reporting"
-- type: markdown
-  attributes:
-    value: |
-      If you find a security issue with our libraries or services [please report it to the Microsoft Security Response Center (MSRC)](https://aka.ms/report-security-issue) with as much detail as possible. Your submission may be eligible for a bounty through the [Microsoft Bounty](http://aka.ms/bugbounty) program. Please do not post security issues to GitHub Issues or any other public site. We will contact you shortly upon receiving the information. We encourage you to get notifications of when security incidents occur by visiting [this page](https://www.microsoft.com/msrc/technical-security-notifications) and subscribing to Security Advisory Alerts.
+    - type: markdown
+      attributes:
+          value: |
+              ## Before Submitting your Issue
+              Our GitHub issues are meant exclusively for bug reports. All issues regarding usage questions will be closed without comment. If you have a quesiton, please check to see if your question is answered in the FAQ for the library you are using:
```

Let's soften the wording here a bit
```suggestion
              Our GitHub issues are meant exclusively for bug reports and feature requests. Usage questions may be closed without comment. If you have a question, please check to see if your question is answered in the FAQ for the library you are using:
```